commons-db/src/main/java/com/palantir/nexus/db/sql/AgnosticResultSet.java

```diff
@@ -20,7 +20,7 @@
 
 public interface AgnosticResultSet {
 
-    List<AgnosticResultRow> rows();
+    List<? extends AgnosticResultRow> rows();
```

Comment:
yeah, I was just noting that that change removed the public modifier on these methods as well and I was surprised that didn't have negative effects.